src/sidebar/panel.js

```diff
@@ -28,7 +28,9 @@ enableSync.setAttribute('title', browser.i18n.getMessage('syncNotes'));
 giveFeedbackButton.setAttribute('title', browser.i18n.getMessage('feedback'));
 giveFeedbackMenuItem.text = browser.i18n.getMessage('feedback');
 giveFeedbackButton.setAttribute('href', SURVEY_PATH);
+giveFeedbackButton.setAttribute('target', '_blank');
```

Comment:
@apoorvaagupta could you please also add `'rel', 'noopener'` besides this new `target` attribute

See https://html.spec.whatwg.org/multipage/links.html#link-type-noopener